### PR TITLE
media_libva_caps_g12: don't report VA_PREDICTION_DIRECTION_FUTURE for hevc scc vdenc

### PR DESCRIPTION
**Pull request by xhaihao**
l0 is equal to l1 for hevc scc vdenc B slice

### CODE REVIEW

**Comment by xhaihao** [comment]
@XinfengZhang could you take a look at this PR ?

File: media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp [hunk 1]
```diff
@@ -518,7 +518,14 @@ VAStatus MediaLibvaCapsG12::GetPlatformSpecificAttrib(VAProfile profile,
         }
         case VAConfigAttribPredictionDirection:
         {
-            *value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_FUTURE | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
+            if ((entrypoint == VAEntrypointEncSliceLP) &&
+                ((profile == VAProfileHEVCSccMain) ||
+                 (profile == VAProfileHEVCSccMain10) ||
+                 (profile == VAProfileHEVCSccMain444) ||
+                 (profile == VAProfileHEVCSccMain444_10)))
+                *value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
```
Comment (Xiaogangli-intel) [inline media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp:526]: @xhaihao Does it mean SCC only support low-delay-B?

Comment (xhaihao) [reply media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp:526]: No, I don't think SCC has this requirement. but the driver requires l0 == l1, see https://github.com/intel/media-driver/blob/master/media_driver/agnostic/gen12/codec/hal/codechal_vdenc_hevc_g12.cpp#L2094-L2148

**Comment by xhaihao** [comment]
See https://github.com/intel/media-driver/pull/1405

**Pull request closed without merging**